evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -20,10 +20,15 @@
                     <tr class="vote-row d-flex">
                         {% if field|is_choice_field %}
                             <td class="vote-question vote-question-choice my-auto{% if field.errors %} choice-error{% endif %}" name="{{ field.name }}">
+                            {{ field.label }}
```

Comment:
the indentation will always be somewhat strange here because the opening tag is inside the if and the closing tag isn't. but could you please still indent this line and those below the opening `<td>` below one level further?

Reply:
:+1:

















⁣